Code/GraphMol/RGroupDecomposition/RGroupCore.cpp

```diff
@@ -54,6 +64,197 @@ void RCore::findIndicesWithRLabel() {
   }
 }
 
+RWMOL_SPTR RCore::extractCoreFromMolMatch(
+    bool &hasCoreDummies, const ROMol &mol, const MatchVectType &match,
```

Comment:
I think in general we have the const refs at the beginning of the function definition in the code.  Is this a style issue anyone cares about?  @greglandrum

Reply:
I just kind of do it automatically, but haven't really thought about it as something important.
Since `hasCoreDummies` is really part of the return value anyway, maybe it makes sense to return an `std::pair<RWMOL_SPTR,bool>` from this function?

Reply:
Now returns `std::pair<RWMOL_SPTR, bool>`